pkg/sriov/network.go

```diff
@@ -418,3 +418,31 @@ func (builder *NetworkBuilder) validate() (bool, error) {
 
 	return true, nil
 }
+
+// CleanAllNetworksByTargetNamespace deletes all networks matched by their NetworkNamespace spec.
+func CleanAllNetworksByTargetNamespace(
+	apiClient *clients.Settings,
+	operatornsname string,
+	targetnsname string,
+	options metaV1.ListOptions) error {
+	networks, err := List(apiClient, operatornsname, options)
+
```

Comment:
Please add glog.V(100).Infof with log what this function is going to perform

Reply:
Added.